### DISCUSSION TITLE
I won't be responding to messages for a couple weeks...

### DISCUSSION THREAD

**Discussion by mmcc-xx** [mmcc-xx/BirdCAGE#46] [Announcements]
I'll be on vacation until the beginning of July.

Happy bird detecting!

**Comment by matthew73210** [comment]
Take care :)

**Comment by grbray** [comment]
@mmcc-xx  Hope you had a good vacation.  Looking to see if we can get some of the PRs merged that fix some issues (Proxy, new docker compose, etc)

**Comment by mmcc-xx** [comment]
Sorry for the delay - vacation and the expected backlog of things at work to deal with. I'll dig into them soon! Thanks for the work!

**Comment by jonbloom** [comment]
@mmcc-xx Hey there. Love this project! It uses the close to the same tech stack as most of my projects I work on both personally and professionally. I forked your repo ([here](https://github.com/jonbloom/birdcage)) and have made a few architectural changes. First, in how the database is interacted with to avoid the need for raw SQL queries using an ORM, [peewee](http://docs.peewee-orm.com/en/latest/). Additionally, I've started work on a VueJS version of the frontend that decouples the frontend code from Flask, reducing the need for multiple Flask apps to run the solution.

I realize that's a pretty big detour from the original repo, but I wanted to get some thoughts to see if you're interested in the work I've started and would be interested in collaborating a bit further on the project. Looking forward to your response!

**Reply by matthew73210** [reply]
This all looks very interesting. Maybe this could become the dev branch / alpha testing?

**Reply by jonbloom** [reply]
Probably, yeah. I want to get somewhat close to feature complete on the frontend before I made a PR.

**Comment by mmcc-xx** [comment]
I'll absolutely take a look - I fully admit to being something of
a dilettante when it comes to this, and I've been treating GPT4 like a
rented mule to help me out. I'd be very happy to make it more performant
and easier to improve and maintain.

**Comment by mmcc-xx** [comment]
I got most of the PRs merged in. The others I have questions about.

**Comment by matthew73210** [comment]
In about a month I come of night shifts, so should be able to have a go at doing some dev stuff. What are you currently working on?
